SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Push.cs

```diff
@@ -21,6 +22,10 @@ namespace Microsoft.AppCenter.Push
     public partial class Push
     {
         private PushNotificationChannel _channel;
+        private static readonly object UWPPushLock = new object();
+
+        // Expose for testing
+        internal string LatestPushToken { get; set; }
```

Comment:
should it be _latestPushToken, check as what;s the format for internal. For private i could see it starts with "_" e.g. "_channel"

Reply:
@guperrot suggested this change in this comment: https://github.com/microsoft/appcenter-sdk-dotnet/pull/1158#discussion_r333793149